labs-ios-starter/Application/Network/ErrorHandler.swift

```diff
@@ -54,7 +54,7 @@ class ErrorHandler {
             NetworkError.unauthorized.rawValue: UserError(title: "Unauthorized", message: "Please login again"),
             NetworkError.forbidden.rawValue: UserError(title: "Forbidden", message: "Access to that resource is restricted"),
             NetworkError.timeout.rawValue: UserError(title: "Timed out", message: "Please check your connection or try again later."),
-            NetworkError.notFound.rawValue: UserError(title: "Not Found", message: "Sorry, we're having trouble finding that resource")
+            NetworkError.notFound.rawValue: UserError(title: "Not Found", message: "Sorry, we're having trouble finding that resource"),
```

Comment:
Hmm.. did the linter add this comma at the end? We should probably fix that rule if so

Reply:
Yeah, thought it was weird, and hated it for a minute, but then thought "oh, it's not causing a compiler error" so I just let it go. If we can get rid of it though, I think it looks cleaner without

Reply:
Just changed it, it's the commas option. It's set to "always", should be "inline"